src/tests/test_utils/common_test_utils/src/node_builders/constant.cpp

```diff
@@ -13,6 +13,13 @@ std::shared_ptr<ov::Node> make_constant(const ov::element::Type& type,
     auto tensor = create_and_fill_tensor(type, shape, in_data);
     return std::make_shared<ov::op::v0::Constant>(tensor);
 }
+
+std::shared_ptr<ov::Node> make_constant(const ov::element::Type& type, const ov::Shape& shape) {
+    InputGenerateData in_data(1, 9);
+    if (type.is_real())
```

Comment:
The `nf4` will return false and this resolution will not be used. Will be OK for tests?